### PR TITLE
Add doc for ISM policy creation delay.

### PR DESCRIPTION
**Pull request by IshaGirdhar**

### CODE REVIEW

**Review by gaurav-05 — APPROVED** [review]

File: content/en/docs/observability/logging/configure-opensearch/opensearch.md [hunk 1]
```diff
@@ -447,7 +447,7 @@ The previous Verrazzano custom resource will generate the following ISM policy.
 
 {{< /clipboard >}}
 
-**NOTE**: The ISM policy created using the Verrazzano custom resource contains a minimal set of configurations. To create a more detailed ISM policy,
+**NOTE**: There could be a delay of few minutes before you can see the policy in OpenSearch. Also, the ISM policy created using the Verrazzano custom resource contains a minimal set of configurations. To create a more detailed ISM policy,
```
Comment (rosemarymarano) [inline content/en/docs/observability/logging/configure-opensearch/opensearch.md:450]: Suggested re-write:
NOTE: There might be a delay of few minutes before you see the policy in OpenSearch. 
(New para, not part of the **NOTE**.)
Also note that the ISM policy created using the Verrazzano custom resource contains a minimal set of configurations. To create a more detailed ISM policy, you can use the OpenSearch REST API. To create a policy using the OpenSearch REST API, do the following:

**Review by rosemarymarano — COMMENTED** [review]
Suggested re-write.

File: content/en/docs/observability/logging/configure-opensearch/opensearch.md [hunk 2]
```diff
@@ -447,8 +447,10 @@ The previous Verrazzano custom resource will generate the following ISM policy.
 
 {{< /clipboard >}}
 
-**NOTE**: There could be a delay of few minutes before you can see the policy in OpenSearch. Also, the ISM policy created using the Verrazzano custom resource contains a minimal set of configurations. To create a more detailed ISM policy,
-you can also use the OpenSearch REST API. To create a policy using the OpenSearch API, do the following:
+**NOTE**: There might be a delay of few minutes before you see the policy in OpenSearch.
```
Comment (rosemarymarano) [inline content/en/docs/observability/logging/configure-opensearch/opensearch.md:450]: -> delay of a few minutes (missing word, *a*)

**Review by rosemarymarano — APPROVED** [review]
Please add the the missing word (a, as noted); otherwise the PR is fine.

**Pull request merged**